.github/workflows/docker-build-release.yaml

```diff
@@ -0,0 +1,39 @@
+name: release-publish
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # GITHUB_REF is tag of the release https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release
+          ref: ${{ github.ref }}
+          # Number of commits to fetch. 0 indicates all history for all branches and tags.
+          fetch-depth: 0
+      - name: Docker meta
```

Comment:
Let's use https://github.com/dev-launchers/strapi/blob/main/.github/workflows/production-image.yaml#L18-L24 for tagging.

Reply:
I think I've done what you requested with this [commit](https://github.com/dev-launchers/site-projects/pull/16/commits/d5161093eb91d72aaf92c5cb4bc8f75b1a4efc13)